src/pages/components/toggle.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { NamespacesConsumer } from 'react-i18next';
+import { withI18next } from '@wapps/gatsby-plugin-i18next';
+import { suomifiTheme } from 'suomifi-ui-components';
+
+import Layout from 'components/layout';
+import SEO from 'components/seo';
+import { Toggle } from 'components/ExampleComponents';
+import ComponentDescription from 'components/ComponentDescription';
+import sideNavData from 'config/sidenav/components';
+import NoteBox from 'components/NoteBox';
+import Section from 'components/Section';
+import ComponentExample from 'components/ComponentExample';
+import { Heading, Text, Paragraph } from 'components/ResponsiveComponents';
+
+let checked = false;
+
+const getAriaLabelText = (
+  state: boolean,
+  label: string,
+  t: Function,
+): string => {
+  return t(`toggle.state.${state ? 'on' : 'off'}`, {
+    name: label,
+  });
+};
+
+const handleClick = (id: string, label: string, t: Function): void => {
+  checked = !checked;
+  document
+    .getElementById(id)
+    .setAttribute('aria-label', getAriaLabelText(checked, label, t));
+};
+
+const getExampleComp = (
```

Comment:
While I'm all for generic methods, this seems a bit complicated for implementing a single example.

Reply:
I agree, it was a bit too complicated for this single example. I have now committed simplified version.